pkg/asset/agent/mirror/registriesconf.go

```diff
@@ -163,7 +174,7 @@ func (i *RegistriesConf) Generate(dependencies asset.Parents) error {
 	i.setMirrorConfig(i.Config)
 
 	if !i.releaseImageIsSameInRegistriesConf(releaseImage.PullSpec) {
-		logrus.Warnf(fmt.Sprintf("The ImageContentSources configuration in install-config.yaml should have at least one source field matching the releaseImage value %s", releaseImage.PullSpec))
+		logrus.Warnf(fmt.Sprintf("The imageDigestSources configuration in install-config.yaml should have at least one source field matching the releaseImage value %s", releaseImage.PullSpec))
```

Comment:
Changes look good. Just one question - previously you had this warning referring to `ImageContentSources or ImageDigestMirrorSet`
You changed it to be just `imageDigestSources` but it would be more accurate to also include `imageContentSources` so the warning is accurate for existing configurations.

Reply:
oh, that's suggested by https://github.com/openshift/installer/pull/6235#discussion_r1151234220 to stop encouraging users to use ImageContentSources.

Reply:
Ah OK thanks.